app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  has_many :posts
```

Comment:
- [ ] According to the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png), post is associated with users table through `author_id` and not `user_id`. This means that the `author_id` column in the posts table holds the foreign key referencing the `id` column in the users table.

```suggestion
  has_many :posts foreign_key: 'author_id'